ci/behat-tags.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+error_reporting(E_ALL);
+ini_set('display_errors', 1);
```

Comment:
Would it be helpful to have a comment here as to why we're changing error reporting?